base/Dockerfile

```diff
@@ -1,10 +1,12 @@
 ARG STACK_TAG=d_latest
 FROM lsstsqre/centos:${STACK_TAG}
 ENV PYTHONUNBUFFERED True
+ARG kafka=python-confluent-kafka
 RUN source /opt/lsst/software/stack/loadLSST.bash \
```

Comment:
You might want to look into the relatively new ["here docs" feature](https://docs.docker.com/reference/dockerfile/#here-documents) of Dockerfiles to improve the readability of this RUN command.  (Make sure to include `set -e` to get the effect of the `&&` command separators.)

Reply:
Do you really need `set -e`? The fourth example shows the command chaining being done on the main line, which seems cleaner.

Reply:
Thanks for the reference.  This is changed to use here documents.